dts/bindings/pinctrl/nuvoton,npcx-pinctrl.yaml

```diff
@@ -69,3 +82,23 @@ child-binding:
         required: false
         type: boolean
         description: Inverse pinmux selection to GPIO
+      psl-in-mode:
+        type: string
+        required: false
+        description: |
+          The assertion detection mode of PSL input selection
+          - "PSL_IN_MODE_LEVEL": Select the detection mode to level detection
+          - "PSL_IN_MODE_EDGE": Select the detection mode to edge detection
+        enum:
+          - "PSL_IN_MODE_LEVEL"
+          - "PSL_IN_MODE_EDGE"
```

Comment:
why not use `lower-case` enums, e.g. `psl-in-mode = "level"`? You can later use `DT_ENUM_IDX` helpers as well.

Reply:
Done.